### DISCUSSION TITLE
连的是阿里云盘，之前都正常使用，今天晚上开始报这个错，是alist的问题还是阿里云的问题：failed get objs: failed to list objs: Post "https://open.aliyundrive.com/adrive/v1.0/openFile/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

### DISCUSSION THREAD

**Discussion by duanzheng** [AlistGo/alist#3786] [Q&A]
failed get objs: failed to list objs: Post "https://open.aliyundrive.com/adrive/v1.0/openFile/list": context deadline exceeded (Client.Timeout exceeded while awaiting headers)

**Comment by anwen-anyi** [comment]
你自己的网络问题

**Reply by duanzheng** [reply]
浏览器不会直连阿里云获取视频列表，这个报错明显是alist和阿里云的通信出了问题

**Comment by duanzheng** [comment]
附上replit的报错信息
![image](https://user-images.githubusercontent.com/4760858/224057163-2d4b1da0-3347-4ecc-bb1a-b8d95d863725.png)
